### PR TITLE
Adding London 8 project examples

### PR DESCRIPTION
**Pull request by kfklein15**
Signed-off-by: KFK <80272258+kfklein15@users.noreply.github.com>

## What does this change?

Module: Final Projects
Week(s): External projects

## Checklist

- [x] I have read the [contributing guidelines](https://syllabus.codeyourfuture.io/contributing/overview)
- [x] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [x] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [x] I have run my code to check it works
- [x] My changes follow our [Style Guide](https://syllabus.codeyourfuture.io/guides/code-style-guide)

## Description

Adding links of app built by London 8 cohort, as examples of what our trainees can achieve

## Who needs to know about this?

 @AliciaE08 @MarianaVazquezCYF

### CODE REVIEW

File: docs/finalproject/external-projects.md [hunk 1]
```diff
@@ -10,9 +10,14 @@ If you are one of these charities, or know one we could work with, you can find
 
 ## Past Projects
 
-Here is an example of a web app we built with an NGO as a Final Project:
+Here are examples of web apps we built as a Final Project:
 
 - [Over The Line with The Compass Collective](https://over-the-line.uk/)
+- [Chatterbox Talents](https://ldn8-horizon-chatterbox-talent.netlify.app/)
+- [Hujreh](https://hujreh.netlify.app/)
```
Comment (SallyMcGrath) [inline docs/finalproject/external-projects.md]:
```suggestion

```
Not a good example

File: docs/finalproject/external-projects.md [hunk 2]
```diff
@@ -10,9 +10,14 @@ If you are one of these charities, or know one we could work with, you can find
...
+- [The Breteau Foundation](https://bugsquashers-edu.netlify.app/)
+- [Pratibha Nakshtra](https://pratibha-nakshtra-teamhapi.netlify.app/)
```
Comment (SallyMcGrath) [inline docs/finalproject/external-projects.md]:
```suggestion

```
Also really unfinished with dummy text. I think we have better examples than this. Can we use the Bikes For Refugees project from GL5, Energiser from WM3,  and Goal from NW4? (Not blocking, as an enhancement

https://linktr.ee/callbackcats - GL5 contacts to get that back up
https://github.com/musayuksel/Goal-app-CYF-final-project 
I wish I could find the WM3 project but I have trawled a few thousand files and I cannot find the WM3 Final Projects tracker anywhere. It's pretty impossible for me to find anything on drive without knowing the exact name or link already as I own all the trainee documents.

They would need to be redeployed on render.com but those are good looking, finished applications and would be a much better example.

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]
Two edits

File: docs/finalproject/external-projects.md [hunk 3]
```diff
@@ -16,6 +16,7 @@ Here are examples of web apps we built as a Final Project:
 - [Chatterbox Talents](https://ldn8-horizon-chatterbox-talent.netlify.app/)
 - [The Breteau Foundation](https://bugsquashers-edu.netlify.app/)
 - [Akwaaba](https://akwaaba.netlify.app/)
+- [Bike for Refugees](https://linktr.ee/callbackcats)
```
Comment (SallyMcGrath) [inline docs/finalproject/external-projects.md]: @kfklein15 this is deployed on Heroku so is down. Everything our trainees built before November 2022 is down. If we put it up, we need to ask the trainees to redeploy on render.com.

**Review by SallyMcGrath — DISMISSED** [review]
I will approve this as I can see there's a lot mounting up.

But you have added a dead link here. Please clean up the dead link  or get the site up?

File: docs/finalproject/external-projects.md (hunk 3 above)
Comment (kfklein15) [reply docs/finalproject/external-projects.md]: Which link is using Heroku? I opened all the links before adding them to the PR and they were all up and running.

**Review by esma-g — CHANGES_REQUESTED** [review]
- The link https://linktr.ee/callbackcats directs is on Heroku and doesn't work: https://cyf-bike-app.herokuapp.com/
- https://bugsquashers-edu.netlify.app/ Lessons button is unresponsive, but maybe it works after registration

**Comment by kfklein15** [comment]
> * The link https://linktr.ee/callbackcats directs is on Heroku and doesn't work: https://cyf-bike-app.herokuapp.com/
> * https://bugsquashers-edu.netlify.app/ Lessons button is unresponsive, but maybe it works after registration

Removed the firs one.
It's ok if the second doesn't work.

**Review by esma-g — APPROVED** [review]
Approved

**Review by illicitonion — APPROVED** [review]

**Pull request merged**